types/node-vault-client/index.d.ts

```diff
@@ -0,0 +1,273 @@
+declare class VaultClient {
+    /**
+     * Boot an instance of Vault
+     *
+     * The instance will be stored in a local hash. Calling Vault.boot multiple
+     * times with the same name will return the same instance.
+     *
+     * @param {String} name - Vault instance name
+     * @param {Object} [options] - options for {@link Vault#constructor}.
+     * @return Vault
```

Comment:
One last comment, but there's no reason to annotate the types for any of this JSDoc; we're already doing type declarations, so these can really only accidentally drift over time.

Reply:
Hi @jakebailey - removed JSDoc annotations